openmls/src/schedule/psk.rs

```diff
@@ -458,3 +464,83 @@ impl From<Secret> for PskSecret {
         Self { secret }
     }
 }
+
+pub(crate) fn load_psks<'p>(
```

Comment:
This function couples two (currently rather unrelated) things: external PSKs (that are "sideloaded") and resumption PSKs (that are handled internally). So, I think the `ResumptionPskStore` is not the correct type that should implement `load_psks`. We could make it so that the `PskSecret` has a `load` method, similar to the previous `new`, but I would prefer to make this change in a follow-up.

Reply:
Ok, let's leave this to a follow-up.